packages/bento-design-system/src/DateField/DateField.tsx

```diff
@@ -25,104 +23,125 @@ type SingleDateFieldProps = {
 type RangeDateFieldProps = {
   type: "range";
   shortcuts?: ShortcutProps<[Date, Date]>[];
-} & FieldProps<[Date | null, Date | null]>;
+} & FieldProps<[Date, Date] | null>;
 type Props = (SingleDateFieldProps | RangeDateFieldProps) & {
-  minDate?: UseDatepickerProps["minBookingDate"];
-  maxDate?: UseDatepickerProps["maxBookingDate"];
-  shouldDisableDate?: UseDatepickerProps["isDateBlocked"];
+  minDate?: Date;
+  maxDate?: Date;
+  shouldDisableDate?: (date: DateValue) => boolean;
```

Comment:
what's `DateValue` here? This was previously `(date: Date) => boolean` (looking at `@datepicker-react/hooks` types), so it's potentially breaking.

Can we find a way to keep it as it was?

Reply:
I forgot to update it to use `Date`